decentraland/unity-explorer
# feat: Scene level lods

# Changes 

Adds scenes-level LODS

# How to test

- Download this LODS assets (https://drive.google.com/file/d/1bZ-J0jHu7iyjA_rliaulAOfv9gyQ4eg3/view?usp=sharing) and put them in the folder `StandaloneWindows64\StandaloneWindows64_Data\StreamingAssets\AssetBundles\lods` (you need to create the folder)
- Enter the world and check that you see scene LODs in the distance
- Walk to the border of the load radius and check that new scenes are loaded (different than the empty ones)
- In debug settings, click the `Toggle LOD color` button. Check that lod color changes
